tests/Datagrid/ProxyQueryTest.php

```diff
@@ -212,4 +214,30 @@ public function testSetHint()
 
         $this->assertEquals(2, $result[0]['id']);
     }
+
+    public function testAddOrderedColumnsCompositeId()
+    {
+        $qb = $this->em->createQueryBuilder()
+                       ->select('IDENTITY(o.id1) as id1, IDENTITY(o.id2) as id2')
+                       ->distinct()
+                       ->from(CompositeIntIdEntity::class, 'o')
+                       ->orderBy('o.id1', 'ASC')
+                       ->addOrderBy('o.id2', 'ASC');
+
+        $pq = $this->getMockBuilder(ProxyQuery::class)
```

Comment:
please `createMock()` here

Reply:
@OskarStark Is that necessary? Support for < PHPUnit 5.4 was dropped in 456fb815935856fffcd4395acffdfdd93d89c6f4

Reply:
createMock is PHPUnit >= 5.4